zebra-state/src/service/arbitrary.rs

```diff
@@ -60,6 +60,30 @@ impl ValueTree for PreparedChainTree {
 pub struct PreparedChain {
     // the proptests are threaded (not async), so we want to use a threaded mutex here
     chain: std::sync::Mutex<Option<(Network, Arc<SummaryDebug<Vec<PreparedBlock>>>)>>,
+    // the height from which to start the chain. If None, starts at the genesis block
+    start_height: Option<Height>,
```

Comment:
Optional: Can we use a `LedgerState` strategy here instead?

It already allows us to set the height and a bunch of other tree properties.